src/scripts/run-tests.sh

```diff
@@ -35,10 +35,11 @@ RunTests() {
         args+=(--config "${PARAM_CONFIG_PATH}")
     fi
     if [[ -n $PARAM_FILES ]]; then
-        files=$(echo "${PARAM_FILES}" | tr "," "\n")
-        for file in $files; do
+        IFS=$'\n'
+        for file in ${PARAM_FILES}; do
```

Comment:
It's defined here:
https://github.com/DataDog/synthetics-test-automation-circleci-orb/blob/3e2ac7c481d0b6daa15a666f09b8291fdc22df7b/src/commands/run-tests.yml#L86
and refers to:
https://github.com/DataDog/synthetics-test-automation-circleci-orb/blob/3e2ac7c481d0b6daa15a666f09b8291fdc22df7b/src/commands/run-tests.yml#L32-L35

We are advertising that we support **glob patterns though**, which can contain commas. Since before we were translating commas into newlines with `tr "," "\n"`, it means we never supported commas in glob patterns, which was an undiscovered bug IMO.

Reply:
In this integration we never defaulted to `{,!(node_modules)/**/}*.synthetics.json` though. We defaulted to an empty string, which was falling back to `datadog-ci`'s default value.

Reply:
Let's document that it supports only newlines, and keep the current implementation which supports only newlines.